test/Microsoft.ML.Tests/Transformers/WordTokenizeTests.cs

```diff
@@ -65,6 +78,24 @@ public void TestOldSavingAndLoading()
                     new WordTokenizingTransformer.ColumnInfo("B", "TokenizeB"),
                 });
             var result = pipe.Fit(dataView).Transform(dataView);
+
+            // Extract the output of the first row (the only row we have because data contains only one TestClass) as a native class.
```

Comment:
```suggestion
            // Extract the transformed result of the first row (the only row we have because data contains only one TestClass) as a native class.
```